draft-ietf-quic-recovery.md

```diff
@@ -206,6 +206,19 @@ Min RTT is the minimum RTT measured over the connection, prior to adjusting by
 ack delay.  Ignoring ack delay for min RTT prevents intentional or unintentional
 underestimation of min RTT, which in turn prevents underestimating smoothed RTT.
 
+### Maximum Ack Delay
+
+QUIC is able to explicitly model delay at the receiver via the ack delay
+field in the ACK frame.  Therefore, QUIC diverges from TCP by calculating a
+MaxAckDelay dynamically, instead of assuming a constant delayed ack timeout
+for all connections.
+
+MaxAckDelay is the maximum ack delay supplied in an all incoming ACK frames.
+MaxAckDelay excludes ack delays that aren't included in an RTT sample because
+they're too large or the largest acked has already been acknowledged.
+MaxAckDelay also excludes ack delays where the largest ack references an
```

Comment:
You've used "largest ack" and "largest acked" here.  Elsewhere in the document, you also use "largest acknowledged."  Might be worth picking one and using it consistently?